src/views/Dashboard.vue

```diff
@@ -106,19 +106,26 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               </v-list-item-subtitle>
             </v-list-item-content>
           </v-list-item>
-          <v-list-item :href=hubUrl>
+          <v-tooltip :disabled=multiUserMode bottom>
+            <template v-slot:activator="{ on }">
+          <div v-on="on" >
+          <v-list-item id="cylc-hub-button" :disabled=!multiUserMode :href=hubUrl>
             <v-list-item-avatar size="60" style="font-size: 2em;">
               <v-icon large>{{ svgPaths.hub }}</v-icon>
             </v-list-item-avatar>
             <v-list-item-content>
-              <v-list-item-title class="title font-weight-light">
-                Cylc Hub
-              </v-list-item-title>
+                <v-list-item-title class="title font-weight-light">
+                  Cylc Hub
+                </v-list-item-title>
               <v-list-item-subtitle>
                 Visit the Hub to manage your running UI Servers
               </v-list-item-subtitle>
             </v-list-item-content>
           </v-list-item>
+          </div>
+            </template>
```

Comment:
The indentation is off (functionally doesn't matter but proper indentation makes it easier to see the HTML hierarchy)

Reply:
Indentation better now, but best to avoid putting tags on the same line like this:
```
<template v-slot:activator="{ on }"> <div v-on="on" >
```

Reply:
@MetRonnie I'm not great at these things so have put it through an auto-prettify.

If you are happy then I think we can merge this.

Reply:
I have no idea why it would want to leave those on the same line. Anyway, it doesn't matter too much